fmc/README.md

```diff
@@ -139,10 +139,11 @@ fields may not be changed or removed). Table revisions with different Major Vers
 | fuse_log_addr         | 4            | ROM        | Fuse Log Address.                                                                                        |
 | rt_dice_pub_key       | 96           | FMC        | RT Alias DICE Public Key.                                                                                |
 | rt_dice_sign          | 96           | FMC        | RT Alias DICE signature.                                                                                 |
+| ldevid_dice_sign      | 96           | FMC        | LDevID DICE signature.                                                                                 |
 | idev_dice_pub_key     | 96           | ROM        | Initial Device ID Public Key.                                                                            |
 | rom_info_addr         | 4            | ROM        | Address of ROMInfo struct describing the ROM digest and git commit.                                      |
 | rtalias_tbs_size      | 2            | FMC        | RT Alias TBS Size.                                                                                      |
-| reserved              | 128          |            | Reserved for future use.                                                                                 |
+| reserved              | 30           |            | Reserved for future use.                                                                                 |
```

Comment:
Should we consider increasing FHT size so that we have more buffer for future needs?

Reply:
Will defer to @jhand2 here.

Reply:
Spoke to Jordan, and we decided to increase the FHT size to 2048, since we already have 2k bytes reserved for the FHT in DCCM. I've updated the docs accordingly.